joomla/joomla-cms
# [4.0] Fixing switcher label for true/false

Pull Request for Issue #26749.

### Summary of Changes
In Global Configuration, `0/1` are saved as `true/false`. Read https://github.com/joomla/joomla-cms/issues/26749#issuecomment-548873881
`false` cast as string returns empty string and `true` returns 1.
Assign it `0` so it can match the option value `0`.
```
			<option value="0">JNO</option>
			<option value="1">JYES</option>
```

### Testing Instructions
Go to Global Configurations
See Site Offline missing label
Apply PR
Make sure switcher labels not missing

Open configuration.php
Toggle Site Offline
Make sure `$offline` is saved as true or false
